test/Microsoft.AspNetCore.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -50,7 +50,7 @@ public BasicTests(MvcTestFixture<BasicWebSite.Startup> fixture)
 #if GENERATE_BASELINES
             ResourceFile.UpdateFile(_resourcesAssembly, outputFile, expectedContent, responseContent);
 #else
-            Assert.Equal(expectedContent, responseContent, ignoreLineEndingDifferences: true);
+            Assert.Equal(expectedContent.Trim(), responseContent, ignoreLineEndingDifferences: true);
```

Comment:
I agree, but it was to make the test pass. `.Trim()` is used frequently in the other tests.